app/policies/course_policy.rb

```diff
@@ -0,0 +1,16 @@
+# Policy for courses controller.
+class CoursePolicy < ApplicationPolicy
+  default_rule :manage?
+
+  def show?
+    true
+  end
+
+  def index?
+    true
+  end
+
+  def manage?
```

Comment:
This policy inherits from ApplicationPolicy which already has the manage? method defined (with the same body) so we can remove this method here.